Doc/library/dis.rst

```diff
@@ -709,12 +709,14 @@ iterations of the loop.
        Exception representation on the stack now consist of one, not three, items.
 
 
-.. opcode:: LOAD_ASSERTION_ERROR
+.. opcode:: LOAD_EXCEPTION_TYPE (type)
```

Comment:
There was a suggestion last year to replace LOAD_ASSERTION_ERROR by a LOAD_COMMON_CONSTANT opcode (which can be used for any const type, not just exceptions).  That could be used to implement https://github.com/faster-cpython/ideas/discussions/202.

Reply:
Yeah, I considered that.

I'd like to leave this as-is, and propose expanding it into a `LOAD_COMMON_CONST` as its own improvement later. I worry that just naming it `LOAD_COMMON_CONST` now will open up the floodgates for PRs adding more "common consts" to the opcode, which will just create lots of bytecode magic version churn, review burden, etc. I'd like to be more principled in deciding what the initial set of constants are (beyond the ones we just really need, like these), using actual data and benchmarks.